scripts/go_install.sh

```diff
@@ -40,6 +40,6 @@ fi
 rm "${GOBIN}/${2}"* || true
 
 # install the golang module specified as the first argument
-go install -tags tools "${1}@${3}"
```

Comment:
I'm not sure why this was needed, pretty sure it was there at the time we had fake go.mod but is no longer needed. I removed it because it was causing errors due to a conflict with a new imported package in kpromo:

```
../../../pkg/mod/github.com/google/certificate-transparency-go@v1.1.2/tools.go:20:2: import "github.com/fullstorydev/grpcurl/cmd/grpcurl" is a program, not an importable package
```

because the above has tag `// +build tools`

Reply:
I agree, probably we either used this script with the fake module or we copied the command from there